labgrid/driver/power/sentry.py

```diff
@@ -0,0 +1,94 @@
+from ..exception import ExecutionError
```

Comment:
Please add a short comment on the top with which device this connector was tested.